File: apprefurbished/src/main/java/com/androidcollider/easyfin/managers/accounts_info/AccountsInfoModule.java
package com.androidcollider.easyfin.managers.accounts_info;

import com.androidcollider.easyfin.repository.Repository;

import dagger.Module;
import dagger.Provides;

/**
 * @author Ihor Bilous
 */

@Module
public class AccountsInfoModule {

    @Provides
    public AccountsInfoManager provideAccountsInfoManager(Repository repository) {
        return new AccountsInfoManager(repository);
    }
}

File: apprefurbished/src/main/java/com/androidcollider/easyfin/common/managers/accounts/accounts_info/AccountsInfoModule.kt
package com.androidcollider.easyfin.common.managers.accounts.accounts_info

import com.androidcollider.easyfin.common.repository.Repository
import dagger.Module
import dagger.Provides

/**
 * @author Ihor Bilous
 */
@Module
class AccountsInfoModule {
    @Provides
    fun provideAccountsInfoManager(repository: Repository): AccountsInfoManager {
        return AccountsInfoManager(repository)
    }
}